dev/prepare-release.sh

```diff
@@ -22,16 +22,16 @@ set -e
 
 if [ -z "$2" ]; then
     cat <<EOF
-Usage: $0 <release-version> <new-development-version-without-SNAPSHOT-suffix>
-Example: $0 1.6.0 1.7.0
+Usage: $0 <release-version> <rc-num>
```

Comment:
Since you recieve the same number of arguments as before but their meaning has changed, I would suggest some kind of sanity check to catch attempts to use it in the old way. For example, if there is a dot in the rc-num, then somebody probably used an old script or old notes to supply the second argument and you can abort the script with a nice error message.